File: app/src/main/java/com/peihou/kotlin/pojo/Student.java
package com.peihou.kotlin.pojo;

import org.greenrobot.greendao.annotation.Entity;
import org.greenrobot.greendao.annotation.Id;
import org.greenrobot.greendao.annotation.Generated;

@Entity
public class Student {
    @Id(autoincrement = true)
    private Long id;
    private String username;
    private int age;
    public int getAge() {
        return this.age;
    }
    public void setAge(int age) {
        this.age = age;
    }
    public String getUsername() {
        return this.username;
    }
    public void setUsername(String username) {
        this.username = username;
    }
    public Long getId() {
        return this.id;
    }
    public void setId(Long id) {
        this.id = id;
    }
    @Generated(hash = 779448599)
    public Student(Long id, String username, int age) {
        this.id = id;
        this.username = username;
        this.age = age;
    }

    public Student(String username, int age) {
        this.username = username;
        this.age = age;
    }

    @Generated(hash = 1556870573)
    public Student() {
    }
}

File: app/src/main/java/com/peihou/kotlin/step5/Student.kt
package com.peihou.kotlin.step5

class Student(name:String,age:Int, var no:String,var score:Int) :Person(name,age) {

}
fun main(args:Array<String>){
    var s=Student("王海东",25,"20131090149",60)
    println("学生：${s.name}")
    println("年龄：${s.age}")
    println("学号：${s.no}")
    println("成绩：${s.score}")
    var sss1:Boolean=false

}